.github/workflows/make-verify.yml

```diff
@@ -0,0 +1,17 @@
+name: make-verify
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  make-verify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup Go environment
+        uses: actions/setup-go@v5.3.0
```

Comment:
does this always deploy go with the version that matches go.mod?

Reply:
It looks like it will always match our go.mod per https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file